academy/xl-run-prod/5-run.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Configure, run and set up a service
+order: 5
+description: Make sure your software is configured and always on
+tag: deep-dive
+---
+
+# Configure, run and set up a service
+
+You have prepared your machine to be part of the upcoming network. Now it is time to:
+
+1. Configure the rest of the software.
+2. Start the software and have it establish a peer-to-peer network with others.
+
+## Configuration
+
+Start by putting `myprojectd` into `/usr/local/bin` or whichever path where you put your executables. Confirm it works with:
+
+```sh
+$ myprojectd version
+```
+
+In the previous section, you already configured some parameters of your node when it comes to Tendermint, in `~/.myprojectd/config/config.toml`.
+
+In `config/app.toml` you will find other parameters to configure. A special note on `halt-height` that assists you in gracefully stopping the node, for instance when applying a migration.
+
+As for the database(s), classic considerations apply. With the `db_dir` flag, consider storing its files in a dedicated and redundant volume. On Linux, you could mount the `data` folder to that separate drive.
+
+Events are also stored in a database, [see `indexer`](https://docs.tendermint.com/master/nodes/configuration.html#indexing-settings), and here too you can choose to store them in a separate one. Note that events are purely a node concern, not a consensus or network one.
+
+TOML and such. Ideas: https://hub.cosmos.network/main/hub-tutorials/join-mainnet.html
```

Comment:
Lines 31-36 feel more like notes than final content (sentence fragments, lack of link wrappers)